managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/AckSetPositionImpl.java

```diff
@@ -31,6 +33,11 @@ public class AckSetPositionImpl implements Position, AckSetState {
     protected final long ledgerId;
     protected final long entryId;
     protected volatile long[] ackSet;
+    @Getter
+    @Setter
+    private int batchMessagesAckedCount;
+    @Getter
+    private boolean positionRemovedFromCursor;
```

Comment:
mutable fields should be `volatile` unless there's other ways to ensure thread safety

Reply:
Fixed